### PR TITLE
Bestest Frontend

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
- Added favicon.
- Added title and description in the body.
- Fixed the description meta tag.
- Searchbox autofocuses on load.
- Changed the default exam selection to "Mid / End Semester".
- Dark mode
- Icons
- And more

### CODE REVIEW

File: frontend/src/components/searchForm.tsx [hunk 1]
```diff
@@ -45,12 +45,12 @@ function CourseSearchForm() {
...
         <div>
           <label for="course">Course Name:</label>
-          <input id="course" value={courseName()} onInput={(e) => setCourseName(e.target.value)} />
+          <input autofocus={true} id="course" value={courseName()} onInput={(e) => setCourseName(e.target.value)} />
         </div>
         <div>
           <label for="exam">Exam:</label>
           <select id="exam" value={exam()} onInput={(e) => setExam(e.target.value)}>
-            <option value="">Select an exam</option>
+            <option value="">Mid / End Semester</option>
```
Comment (shikharish) [inline frontend/src/components/searchForm.tsx:53]: Just "Any" would be better.

Comment (chirag-ghosh) [reply frontend/src/components/searchForm.tsx:53]: All is better than Any _if_ we are changing

Comment (harshkhandeparkar) [reply frontend/src/components/searchForm.tsx:53]: ready to change anything but _should i_?

File: frontend/index.html [hunk 2]
```diff
@@ -4,8 +4,9 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <title>Question Paper Search - Metakgp</title>
-    <description>Search for question papers when library can't save you.</description>
+    <meta name="description" content="Search for question papers when the library can't save you." />
```
Comment (chirag-ghosh) [inline frontend/index.html:7]: While you are at it, please add the other meta tags

Comment (harshkhandeparkar) [reply frontend/index.html:7]: I am no longer at it. Let's do this in #16.

**Review by chirag-ghosh — APPROVED** [review]

**Pull request merged**